stories/browser/src/UIKeySheet.tsx

```diff
@@ -30,9 +42,21 @@ function UIKeySheetContent({
         setTimeout(focus, 300);
     }, []);
 
+    const onSend = React.useCallback(() => {
+        onKeyRetrieved(key);
+    }, [key]);
+
     return (
         <View style={styles.wrapper}>
-            <UIKeyTextView ref={keyRef} label={label} onDone={onKeyRetrieved} />
+            <UIKeyTextView
+                ref={keyRef}
+                label={label}
+                onDone={onKeyRetrieved}
+                onSuccess={setSuccess}
+                onError={setError}
+                onChangeKey={setKey}
+            />
+            <UIBoxButton onPress={onSend} disabled={!success || error} title={buttonTitle} />
```

Comment:
`!success || error` result could be memoized

Reply:
https://github.com/tonlabs/UIKit/pull/1476/commits/c1c400bc4e4e15c9d33964fd64633a8ca03fedd4